### PR TITLE
Setup Express

### PR DESCRIPTION
**Pull request by armaanchhina**
This PR sets up an Express server and configures a linter for the project.

The Express server is configured to respond with "Hello World!" when accessing http://localhost:3000.

ESLint has been added to enforce coding standards and ensure code quality.
Run the Server: npm run start
Navigate to http://localhost:3000 in your browser to see the "Hello World!" message.

Run the Linter: npm run lint
This will check the codebase for linting issues.

### CODE REVIEW

File: package-lock.json
Comment (maroar) [inline package-lock.json:1]: I think that we do not need this file.

Comment (armaanchhina) [reply package-lock.json:1]: Why do we not need this file?

Comment (maroar) [reply package-lock.json:1]: nevermind... I think we can go with this file... my bad ^^'

**Review by maroar — APPROVED** [review]

**Review by TylerWon — COMMENTED** [review]
I think we should move all the backend-related files to its own folder so it doesn't conflict with the frontend files

**Review by TylerWon — APPROVED** [review]

**Review by maroar — APPROVED** [review]

**Pull request merged**